Source/Core/InputCommon/ControllerInterface/evdev/evdev.cpp

```diff
@@ -665,7 +693,7 @@ evdevDevice::~evdevDevice()
   {
     m_input_backend.RemoveDevnodeObject(node.devnode);
     libevdev_free(node.device);
-    close(node.fd);
+    m_input_backend.DeferredClose(node.fd);
```

Comment:
Are these `fd`s going to remain open until `CleanUp` happens to be called in `PopulateDevices` or immediately after `AddDeviceNode`? They could even leak on shutdown, unless I am missing something? Maybe `CleanUp` should be called every iteration of the `HotplugThreadFunc` loop?

Reply:
No longer deferred in the latest code.